ansible_collections/arista/avd/roles/eos_designs/defaults/main/default-templates.yml

```diff
@@ -82,3 +82,51 @@ default_templates:
       connected_endpoints_ethernet_interfaces: 'interface_descriptions/connected_endpoints/ethernet-interfaces.j2'
       connected_endpoints_port_channel_interfaces: 'interface_descriptions/connected_endpoints/port-channel-interfaces.j2'
       overlay_loopback_interface: 'interface_descriptions/loopback_interfaces/overlay-loopback.j2'
+
+  l2ls:
+    structured_config:
+      # Render Structured Configuration
+      # Base features
+      - template: "base/main.j2"
+      # MLAG feature
+      - template: "mlag/main.j2"
+      # Underlay feature
+      - template: "underlay/main.j2"
+      # Overlay feature
+      # - template: "overlay/main.j2"
+      # Core Interfaces Feature
+      - template: "core_interfaces/main.j2"
+      # L3 Edge feature
+      - template: "l3_edge/main.j2"
+      # Tenants feature
+      - template: "network_services/main.j2"
+      # Connected Endpoints feature
+      - template: "connected_endpoints/main.j2"
+      # Inband Management feature
+      - template: "inband_management/main.j2"
+      # Merge custom_structured_configuration last
+      - template: "custom_structured_configuration/main.j2"
+        options:
+          list_merge: "{{ custom_structured_configuration_list_merge }}"
+          strip_empty_keys: false
+    ip_addressing:
+      router_id: 'ip_addressing/router-id.j2'
+      ipv6_router_id: 'ip_addressing/router-id-ipv6.j2'
+      mlag_ip_primary: 'ip_addressing/mlag-ip-primary.j2'
+      mlag_ip_secondary: 'ip_addressing/mlag-ip-secondary.j2'
+      mlag_l3_ip_primary: 'ip_addressing/mlag-l3-ip-primary.j2'
+      mlag_l3_ip_secondary: 'ip_addressing/mlag-l3-ip-secondary.j2'
```

Comment:
@tgodaA - I think so...  In many of the L2LS implementations I have seen, customers will peer on a separate SVI for an IGP peering.  This would be done from the `mlag_peer_l3_ipv4_pool:`

Reply:
Yes, IPs will be used on the spines.